aamt_parse2.pm

```diff
@@ -355,16 +605,17 @@ sub pars_UploadPublishSettingsAllSites
         ilog_setLogInformation('INT_INFO',$recoveryFile,MSG_FILE_OPEN,'');
     }
 
-    ilog_print(1,"\n\n");
-    ui_printline();
-    ilog_printf(1,"Please enter the location of the publish settings: \n");
-    ui_printline();
+#    ilog_print(1,"\n\n");
```

Comment:
You need to add a cmd line flag for running in unattended/non-interactive mode. If this flag is true, then you can get the information from the cmd line, else keep the current logic of asking the user for input.